ITISFoundation/osparc-simcore
# is1047/remove-minio-and-webclient

## What do these changes do?

- [x] integrate new qooxdoo-kit into workflow (see ``services/web/client/README.md``)
   - cleanup of web/client folder and new doc
   - remove web_client service
- [x] simplified main Makefile by splitting build options

## Related issue number
implementing issue #1047 

## How to test
Basically it should work as before 

     $ make build-devel
     $ make up-devel  # (now this is IDENTICAL to make up-webclient-devel
check website, modify webclient code, ... refresh

     $ make build up-prod

**NOTICE**: ``up-``**prod**

@odeimaiz  please test if everything works fine in windows
@odeimaiz, @ignapas  please make sure you are comfortable in dev mode for the front-end (please see [README](services/web/client/README.md) and [Makefile](services/web/client/Makefile) for more details and alternatives

## Checklist
